course/layouts/shortcodes/resource_link.html

```diff
@@ -1,5 +1,5 @@
 {{- $uuid := index .Params 0 -}}
 {{- $title := index .Params 1 -}}
 {{- range where $.Site.Pages "Params.uid" $uuid -}}
-    <a href="{{- .Permalink -}}">{{ $title }}</a>
+    <a class="d-block" href="{{- .Permalink -}}">{{ $title }}</a>
```

Comment:
tbh, I'm not liking this solution as it will cause the resource links in paragraphs/texts to be displayed in an entirely new line. 
I spent some time but couldn't figure out something better. Maybe we can make some customizations in the editor to render a new line **only** when resource links are placed/created in different lines.
Perhaps, @gumaerc @alicewriteswrongs can share their thoughts

Reply:
yes I think you're right, we don't want to introduce a solution to one line-break issue that creates another one

Reply:
I agree, we shouldn't be setting these links to be block level elements because it will cause unintended line breaks when the goal here is to solve unintentionally _not_ inserting line breaks.  @alicewriteswrongs noted on the original issue that a potential short term solution would be to instruct content editors to put a resource link inside a list if it's by itself, even if it's only one link.  This would work, and the only way I can think of that might be better would be if we can somehow recognize that a resource link is on a line by itself in CKEditor's turndown rule, maybe by looking at adjacent children of the node or something.  If we could do that, we could add an optional boolean argument to `resource_link.html` called `inline` or something that defaults to `true`.  If we could catch that the resource link is being rendered on a line by itself, we could set that to `false` and then intentionally render a line break in that case?

Reply:
@gumaerc Yeah, I also thought about this approach but there were some use-cases in which this approach might not be suitable.